mne/io/egi/tests/test_egi.py

```diff
@@ -288,4 +291,63 @@ def test_io_egi_crop_no_preload():
     assert_allclose(raw._data, raw_preload._data)
 
 
+@requires_version('mffpy', '0.5.7')
+@requires_testing_data
+@pytest.mark.parametrize('idx, cond, tmax, signals, bads', [
+    (0, 'Category 1', 0.016, egi_txt_evoked_cat1_fname,
+     ['E8', 'E11', 'E17', 'E28', 'ECG']),
+    (1, 'Category 2', 0.0, egi_txt_evoked_cat2_fname,
+     ['E257', 'EMG'])
+])
+def test_io_egi_evokeds_mff(idx, cond, tmax, signals, bads):
+    """Test reading evoked MFF file."""
+    # Test reading all conditions from evokeds
+    evokeds = read_evokeds_mff(egi_mff_evoked_fname)
+    assert len(evokeds) == 2
+    # Test reading list of conditions from evokeds
+    evokeds = read_evokeds_mff(egi_mff_evoked_fname, condition=[0, 1])
+    assert len(evokeds) == 2
+    # Test invalid condition type
+    with pytest.raises(TypeError):
```

Comment:
Help both make clearer what's being tested, and more specific/targeted
```suggestion
    with pytest.raises(TypeError, match='some matching substring'):
```